cmd/packer-sdc/internal/renderdocs/docs-partials/packer-plugin-sdk/multistep/commonsteps/HTTPConfig-not-required.mdx

```diff
@@ -7,14 +7,19 @@
   started. The address and port of the HTTP server will be available as
   variables in `boot_command`. This is covered in more detail below.
 
-- `http_content` (map[string]string) - Key/Values to serve using an HTTP server. http_content works like and
-  conflicts with http_directory The keys represent the paths and the values
-  contents. This is useful for hosting kickstart files and so on. By
+- `http_content` (map[string]string) - Key/Values to serve using an HTTP server. `http_content` works like and
+  conflicts with `http_directory`. The keys represent the paths and the
+  values contents, the keys must start with a slash, ex: `/path/to/file`.
+  `http_content` is useful for hosting kickstart files and so on. By
   default this is empty, which means no HTTP server will be started. The
   address and port of the HTTP server will be available as variables in
-  `boot_command`. This is covered in more detail below. Example: Setting
-  `"foo/bar"="baz"`, will allow you to http get on
-  `http://{http_ip}:{http_port}/foo/bar`.
+  `boot_command`. This is covered in more detail below.
+  Example:
+   ```hcl
+  .  http_content = {
+     "/a/b" = "content
+   }
```

Comment:
I'd like an example more close to the reality, as the content will most of the time be in a file itself. 
```suggestion
  .  http_content = {
     "/a/b" = file("http/b")
   }
```

If you'd like to keep it that way - it's missing the last `"`
```suggestion
  .  http_content = {
     "/a/b" = "content"
   }
```

Reply:
Ah good idea, I like the first example, also adding a templatefile example.